### PR TITLE
Enable source map generation for ts/js files (#1711)

### LINKED ISSUE #1711: Enable source maps for minified js files
We're using build web compiler 2022 to minify js results of typescript compiler in the following areas:

1- src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/wwwroot/scripts
2- src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Scripts/app.ts
3- src/Client/Web/Bit.Client.Web.BlazorUI/Scripts/bit.blazorui.ts

The toolings current configuration generates no source map for the minified file which results into poor developer experience while debugging our typescript codes

### PR DESCRIPTION
**Pull request by RezaKargar**
This closes #1711

### CODE REVIEW

**Comment by RezaKargar** [comment]
@ysmoradi Please checkout this PR

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/wwwroot/scripts/todoTemplate.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@
                 return unescape(trim(cookie[1]));
             }
         }
-        return null;
+        return "";
```
Comment (msynk) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/wwwroot/scripts/todoTemplate.ts:25]: plz revert this change

**Review by msynk — CHANGES_REQUESTED** [review]
the minification is done by the WebCompiler, I don't think that enabling the sourcemap option in tsconfig file has any effect on this issue.
@ysmoradi any idea?

**Comment by ysmoradi** [comment]
> the minification is done by the WebCompiler, I don't think that enabling the sourcemap option in tsconfig file has any effect on this issue. @ysmoradi any idea?

This won't solve the issue. We need to do something beyond typescript settings

**Comment by msynk** [comment]
since this is not the solution, I'm closing this PR.

**Pull request closed without merging**